postgres/datadog_checks/postgres/statements.py

```diff
@@ -311,20 +304,23 @@ def _explain_new_pg_stat_activity(self, samples, seen_statements, seen_statement
         return events
 
     def _collect_execution_plans(self, instance_tags):
+        if not self._activity_sample_rate_limiter:
+            self._activity_sample_rate_limiter = ConstantRateLimiter(self.config.collect_exec_plans_rate_limit)
```

Comment:
Why lazy loaded?

Reply:
we don't have `self.config` until after the constructor is done

Reply:
Ah fair enough 👍